imagespace/README.md

```diff
@@ -41,8 +42,11 @@ tangelo --port 9220
 ### Girder with imagespace plugin
 
 1. First ensure you have the proper [Girder_prerequisites](http://girder.readthedocs.org/en/latest/prerequisites.html),
+ - To build on MacOS you will need Xcode if you don't have it already. https://developer.apple.com/xcode/
+ - Check if `xcode-select -p` is pointing to Xcode app Developer directory else point xcode-select to the Xcode app Developer directory using the following command: `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`
+ - Use latest version of node 6.x 
```

Comment:
I expected these to be there in Girder prerequisite but I can't find both suggestions there . http://girder.readthedocs.io/en/latest/prerequisites.html#os-x

- `brew install node` installs 7.x version of node which didn't worked for me and I had to install 6.x
- I had to debug xcode issue myself. I think some systems come with xcode installed so people might not have faced this issue. But I had to do it myself.

If you can push these upstreams it will be great!